packages/nc-gui/components/smartsheet/Cell.vue

```diff
@@ -61,7 +62,7 @@ interface Props {
 
 const props = defineProps<Props>()
 
-const emit = defineEmits(['update:modelValue', 'save', 'navigate', 'update:editEnabled'])
+const emit = defineEmits(['update:modelValue', 'save', 'navigate', 'update:editEnabled', 'update:value'])
```

Comment:
Why `'update:value'` and `'update: modelValue'` ? Any reason to keep both ?

Reply:
removed 'update:value'

Reply:
'update:value' is emitted when the column is updated. 
'update:modelValue' is emitted when the cell value is updated

Reply:
Included the `update:value` as it is emitted whenever a column property is changed(referred other files). 
Is it required? I checked and we don't have a receiver for it